bin/clean-dir.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * example package.json entry: 
+ * "prebuild": "node ./node_modules/@iobroker/adapter-dev/bin/clean-dir.js ./build",
```

Comment:
`clean-dir.js` should be added to `adapter-dev`s package.json `bin`, like the existing tools. That way the call on the developer side would look like this:
```suggestion
 * "prebuild": "clean-dir ./build",
```